js/renovation/preact_wrapper/__tests__/data_grid.test.tsx

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable class-methods-use-this */
+import DataGridComponent from '../data_grid';
+import BaseComponent from '../component';
+
+jest.mock('../component', () => {
+  class T {
+    static beginUpdateMock = jest.fn();
+
+    static endUpdateMock = jest.fn();
```

Comment:
Move mock functions outside jest.mock. In this case mock functions should have prefix 'mock'. It will be simpler to access the mock functions in the tests and you can use mockImplementation instead of class T.